src/DotNetLightning.Core/Channel/Commitments.fs

```diff
@@ -88,6 +88,30 @@ type WaitingForRevocation = {
             (fun w -> w.ReSignASAP),
             (fun v w -> { w with ReSignASAP = v })
 
+type RemoteNextCommitInfo =
+    | WaitingForRevocation of WaitingForRevocation
```

Comment:
It is good to have a distinct identifier for DU case and the type of data inside that case.
Since you don't have to match it by full name in that way.
In this case I think
```
| Waiting of WaitingForRevocation
```

is better.

Reply:
That's a good point. I've changed it.